README.md

```diff
@@ -1,40 +1,131 @@
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
+
 # openui5-basic-template-app
-This is a basic app template which you can use as a starting point to build OpenUI5 apps.
+> [OpenUI5](https://github.com/SAP/openui5) basic template app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
+
+## Prerequisites
+- The **UI5 CLI** of the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).
+    - For installation instructions please see: [Installing the UI5 CLI](https://github.com/SAP/ui5-tooling#installing-the-ui5-cli).
 
 ## Getting started
+1. Clone the repository (on the `master` branch) and navigate into it
+    ```sh
+    git clone https://github.com/SAP/openui5-basic-template-app.git
+    cd openui5-basic-template-app
+    ```
+1. Install all dependencies
+    ```sh
+    npm install
+    ```
+
+1. Start a local server and run the application (http://localhost:8080/index.html)
+    ```sh
+    ui5 serve -o /index.html
+    ```
+
+## Testing
+* Run ESLint code validation
+    ```sh
+    npm run lint
+    ```
+* Start a local server and execute the tests automatically after every change
+    ```sh
+    npm run watch
+    ```
+* Run ESLint, start a local server and run the tests in CI mode
+    ```sh
+    npm test
+    ```
+## Building
+### Option 1: Standard preload build
+1. Execute the build
+    ```sh
+    ui5 build -a
+    ```
+1. Run the result
```

Comment:
remove everything starting with this line and refer to the build tools documentation:

For more build and development options please see: [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling)